doc/modules/mod_amp.md

```diff
@@ -16,13 +13,12 @@ none
 ### Example Configuration
 
 ```
-  {mod_amp, []},
+  {mod_amp, []}
 ```
 
 ### XEP Support
 
-What follows is a short description of which parts of the XEP-0079 specification
-mod_amp supports.
+What follows is a short description of which parts of the XEP-0079 specification mod_amp supports.
```

Comment:
Here is a short description of the parts of XEP-0079 specification that are supported by mod_amp.